### PR TITLE
Reduce printing

### PR DESCRIPTION
**Pull request by idohalevi**

### CODE REVIEW

File: driver.go [hunk 1]
```diff
@@ -256,13 +258,29 @@ func getEnvDuration(env string, dValue time.Duration) time.Duration {
...
-			logrus.Info(fmt.Sprintf("Using default drain timeout %v\n", dValue))
-			retDuration = defaultLogsDrainTimeout
+			logrus.Info(fmt.Sprintf("Using default drain timeout %+v\n", dValue))
+			return defaultLogsDrainTimeout
 		}
 	}
 	return retDuration
 }
 
+func getEnvBool(env string, dValue bool) bool {
```
Comment (yyyogev) [inline driver.go]: whats this for?

File: driver.go [hunk 2]
```diff
@@ -256,13 +258,29 @@ func getEnvDuration(env string, dValue time.Duration) time.Duration {
...
+	// Getenv retrieves the value of the environment variable named by the key.
+	// It returns the value, which will be empty if the variable is not present.
+	eVal := os.Getenv(env)
+	if eVal != "" {
+		return dValue
+	}
+
+	retVal, err := strconv.ParseBool(eVal)
```
Comment (yyyogev) [inline driver.go]: are you parsing an empty string to bool?

File: driver.go [hunk 3]
```diff
@@ -256,13 +258,29 @@ func getEnvDuration(env string, dValue time.Duration) time.Duration {
...
 		}
 	}
 	return retDuration
 }
 
+func getEnvBool(env string, dValue bool) bool {
+	// Getenv retrieves the value of the environment variable named by the key.
+	// It returns the value, which will be empty if the variable is not present.
+	eVal := os.Getenv(env)
+	if eVal != "" {
```
Comment (yyyogev) [inline driver.go]:
```suggestion
	if eVal == "" {
```

File: driver.go [hunk 4]
```diff
@@ -271,8 +289,14 @@ func newLogzioSender(loggerInfo logger.Info, token string, sender *logzio.Logzio
...
 	eDiskThreshold := getEnvInt(envDiskThreshold, defaultDiskThreshould)
+
+	debugWriter := os.Stderr
+	if debug := getEnvBool(envDebug, defaultDebug); !debug {
+		debugWriter = nil
+	}
+
 	lsender, err := logzio.New(token,
-		logzio.SetDebug(os.Stderr),
+		logzio.SetDebug(debugWriter),
```
Comment (yyyogev) [inline driver.go]: what will happen when debugWriter == nil?

File: driver.go [hunk 5]
```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/docker/docker/daemon/logger"
 	"github.com/docker/docker/daemon/logger/jsonfilelog"
 	"github.com/docker/docker/daemon/logger/loggerutils"
-	"github.com/dougEfresh/logzio-go"
```
[the remark this answers is not in the document]
Comment (idohalevi) [reply driver.go]: A customer who wrote the original `logzio-go`

File: driver.go (hunk 1 above)
Comment (idohalevi) [reply driver.go]: Parsing environment variable (string) to its boolean value

File: driver.go (hunk 3 above)
Comment (idohalevi) [reply driver.go]: Thanks

File: driver.go (hunk 4 above)
Comment (idohalevi) [reply driver.go]: `logzio-go` won't print any [debug](https://github.com/logzio/logzio-go/blob/master/logziosender.go#L328)

**Pull request merged**